docs/schema.md

```diff
@@ -73,7 +73,11 @@ A JSON Schema for validation is also available
 		"package": {
 			"ecosystem": string,
 			"name": string,
-			"purl": string
+			"purl": string,
+			"severity": [ {
```

Comment:
We should pull this one level up out of the `"package"` field into a field under "affected". The "package" field is meant for identifying the package. Any other properties go alongside it, not inside.